gitbook-docs/market/user-guides/circuit-developer.md

```diff
@@ -1,51 +1,64 @@
-# Circuit Developer
+# Circuit developer
 
-This is an entity that prepares circuits for the proof market. The most efficient way to create a circuit definition for Proof Market is [zkLLVM](https://github.com/NilFoundation/zkllvm). All dependencies and build instructions are inside the [zkLLVM's repository](https://github.com/NilFoundation/zkllvm).
+This is an entity that prepares circuits for Proof Market.
+The most efficient way to create a circuit definition for Proof Market is
+[zkLLVM](https://github.com/NilFoundation/zkllvm) circuit compiler.
+You can find the required dependencies and build instructions in the
+[zkLLVM's repository](https://github.com/NilFoundation/zkllvm).
 
-Anyone can generate circuits. They are serialised & published on the proof market. This allows the reuse of the circuits by all other proof requesters.&#x20;
+Anyone can generate circuits.
+They are serialized & published on Proof Market, allowing reuse of the circuits by
+all proof requesters.
 
-## Create a new circuit (using [zkLLVM](https://github.com/NilFoundation/zkllvm))
+## Create a new circuit using [zkLLVM](https://github.com/NilFoundation/zkllvm)
 
-```
+```bash
 make -C ${ZKLLVM_BUILD:-build} <circuit target name> -j$(nproc)
 ```
 
 ## Prepare a statement with a circuit description for Proof Market
 
-Circuits are stored as a _statement_ structure on Proof Market. Statement description example can be found in /example directory
+Circuits are stored as a _statement_ structure on Proof Market.
+Example of a statement description can be found in the `/example` directory.
 
-```
+```bash
 python3 scripts/prepare_statement.py -c <zkllvm output> \
--o <statement description file> \ 
--n <statement name> -t <circuit type>
+    -o <statement description file> \ 
+    -n <statement name> -t <circuit type>
 ```
 
-Provide the necessary information listed in the output statement file
+Make sure to provide the necessary information listed in the output statement file.
```

Comment:
I don't understand what I should provide. What's necessary?

Reply:
I wish I knew :)

I think this sentence is about circuit's parameters which are actually the statement_tools' parameters:
    --circuit <zkllvm output> \
    --type <circuit type> \
    --output <statement description file> \
    --name <statement name> \
    --private | --public

So I guess this sentence is redundant here?